package.xml

```diff
@@ -13,10 +13,12 @@
   <buildtool_depend>rosidl_parser</buildtool_depend>
 
   <build_depend>pkg-config</build_depend>
+  <build_depend>python3-yaml</build_depend>
   <build_depend>rclcpp</build_depend>
   <build_depend>rmw_implementation_cmake</build_depend>
   <build_depend>std_msgs</build_depend>
 
+  <exec_depend>python3-yaml</exec_depend>
```

Comment:
If it's only used during the building of the bridge and the ros1_bridge python package isn't needed at runtime then we can remove this exec_depend.